modules/logging-release-notes-6-0-0.adoc

```diff
@@ -0,0 +1,97 @@
+
+* With this release, an alert triggers when log collectors buffer logs to a node's file system and use over 15% of the available space. This may indicate the log collectors are facing back pressure from their outputs, and administrators should act to prevent potential issues on the cluster node. (link:https://issues.redhat.com/browse/LOG-5381[LOG-5381])
+
+* This enhancement updates the selectors for all components to use common Kubernetes labels. (link:https://issues.redhat.com/browse/LOG-5906[LOG-5906])
+
+* With this update, the collector configuration will deploy as a ConfigMap instead of a Secret. This change allows users to view and edit the configuration when the ClusterLogForwarder is set to `Unmanaged`.(link:https://issues.redhat.com/browse/LOG-5599[LOG-5599])
+
+* With this release, the log level for the Vector collector can be configured using an annotation on the ClusterLogForwarder. The log level options include trace, debug, info, warn, error, or off. (link:https://issues.redhat.com/browse/LOG-5372[LOG-5372])
+
+* This enhancement adds validation to reject configurations where CloudWatch outputs use multiple AWS roles. Previously, logs were incorrectly routed to the first AWS role defined. (link:https://issues.redhat.com/browse/LOG-5640[LOG-5640])
+
+* This enhancement removes the *Log Bytes Collected* and *Log Bytes Sent* graphs from the metrics dashboard. (link:https://issues.redhat.com/browse/LOG-5964[LOG-5964])
+
+* With this release, the must-gather functionality is updated to only capture information for inspecting Logging 6.0 components, including Vector deployments from `ClusterLogForwarder.observability.openshift.io` resources and the Red Hat managed `LokiStack` provided by the Loki Operator. (link:https://issues.redhat.com/browse/LOG-5949[LOG-5949])
+
+=== Log Storage
+
+* With this release, the responsibility for deploying the logging view plugin shifts from the Cluster Logging Operator to the Cluster Observability Operator. For new log storage installations that need visualization, the Cluster Observability Operator and the associated UIPlugin resource must be deployed. Refer to the official product documentation for more details. (link:https://issues.redhat.com/browse/LOG-5461[LOG-5461])
+
+* With this update, the Azure storage secret validation now provides early warnings for specific error conditions. (link:https://issues.redhat.com/browse/LOG-4571[LOG-4571])
+
+[id="logging-release-notes-6-0-0-technology-preview-features"]
+== Technology Preview features
+
+* This release introduces a Technology Preview feature for log forwarding using OpenTelemetry. A new output type,` OTLP`, allows sending JSON-encoded log records using the OpenTelemetry data model and resource semantic conventions. (link:https://issues.redhat.com/browse/LOG-4225[LOG-4225])
+
+[id="logging-release-notes-6-0-0-known-issues"]
+== Known issues 
+
+None.
+
```

Comment:
Is it normal in Logging docs to say "None." in the event there's no known issues? If there are no Known Issues, I think this can just be removed.